.travis.yml

```diff
@@ -19,6 +19,7 @@ cache:
 jobs:
   include:
     - stage: Tests
+      if: NOT branch =~ /^it\/.*$/)
```

Comment:
* If somebody decides to use it/* in the base repo as location for his/her initial pull request, s?he will not get the unit tests. Not that cool IMHO. I personally prefer not to use pull requests like this, but maybe colleagues ...

* Should we take the repo from that the PR is opened into account? With that setup I do not get unit tests when I propose from a fork from a branch like `it/*`

Reply:
Forget about my comment. In principle valid, but whining at a high level ...